openssl-sys/build.rs

```diff
@@ -164,6 +168,13 @@ fn try_pkg_config() {
     // Otherwise if we're going to windows we probably can't use pkg-config.
     if target.contains("windows-gnu") && host.contains("windows") {
         env::set_var("PKG_CONFIG_ALLOW_CROSS", "1");
+    } else if target.contains("windows-gnu") && host.contains("linux") {
```

Comment:
Hmm, yes, you're right. Actually I deem that manually setting `OPENSSL_LIB_DIR` would be sufficient. Better to leave this to the users.